src/sentry/models/options/option.py

```diff
@@ -1,10 +1,24 @@
+from enum import Enum
+
 from django.db import models
 from django.utils import timezone
 
 from sentry.db.models import Model, control_silo_only_model, sane_repr
 from sentry.db.models.fields.picklefield import PickledObjectField
 
 
+class OptionsTypes(Enum):
+    """Enumerated type referring to potential sources of Sentry Options"""
+
+    LEGACY = "legacy"
+    AUTOMATOR = "automator"
+    CLI = "cli"
```

Comment:
Should we have an entry for the killswitch system as well ?
Not a blocking concern

Reply:
We definitely should. I'll add one in there.